### ISSUE TITLE
Add exponential moving average to vega-lite

### ISSUE THREAD

**Issue by julieg18** [vega/vega-lite#9200]
## Enhancement Description

`aggregate_params` was added to vega aggregation transforms in https://github.com/vega/vega/pull/3686, allowing exponential moving average. Can we add a way to add these `aggregation_params` into vega-lite aggregation transforms? 

I'm new to vega and vega-lite, but, looking at the code, I think I'd be able to help in adding this into vega-lite. 

## Checklist

- [x] I checked for duplicate issues.

**Comment by domoritz** [comment]
Absolutely. Please send a pull request. I already added the types in https://github.com/vega/vega-lite/commit/e3a6232e0fee0802edbbc5cf4180c359ea72f1bf but we need the new aggregate params argument as well.
